cmd/minikube/cmd/config/config.go

```diff
@@ -46,122 +49,145 @@ type Setting struct {
 		set:         SetInt,
 		validations: []setFn{IsPositive},
 	},
 	{
 		name:        "cpus",
+		description: "Number of CPUs allocated to each minikube node",
 		set:         SetInt,
 		validations: []setFn{IsValidCPUs},
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
 		name:        "disk-size",
+		description: "Disk size allocated to the minikube VM (format: <number>[<unit>], where unit = b, k, m or g). (default \"20000mb\")",
 		set:         SetString,
 		validations: []setFn{IsValidDiskSize},
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
 		name:        "host-only-cidr",
+		description: "The CIDR to be used for the minikube VM (virtualbox driver only) (default \"192.168.99.1/24\")",
 		set:         SetString,
 		validations: []setFn{IsValidCIDR},
 	},
 	{
 		name:        "memory",
+		description: "Amount of RAM to allocate to Kubernetes (format: <number>[<unit>], where unit = b, k, m or g).",
 		set:         SetString,
 		validations: []setFn{IsValidMemory},
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
 		name:        "log_dir",
+		description: "If non-empty, write log files in this directory",
 		set:         SetString,
 		validations: []setFn{IsValidPath},
 	},
 	{
-		name: "kubernetes-version",
-		set:  SetString,
+		name:        "kubernetes-version",
+		description: "The Kubernetes version that the minikube VM will use (ex: v1.2.3, 'stable' for v1.20.7, 'latest' for v1.22.0-alpha.2). Defaults to 'stable'.",
+		set:         SetString,
 	},
 	{
 		name:        "iso-url",
+		description: fmt.Sprintf("Locations to fetch the minikube ISO from. (default %s)", download.DefaultISOURLs()),
 		set:         SetString,
 		validations: []setFn{IsValidURL, IsURLExists},
 	},
 	{
-		name: config.WantUpdateNotification,
-		set:  SetBool,
+		name:        config.WantUpdateNotification,
+		description: "If true, will notify on start if there's a newer stable version of minikube available.",
+		set:         SetBool,
 	},
 	{
-		name: config.WantBetaUpdateNotification,
-		set:  SetBool,
+		name:        config.WantBetaUpdateNotification,
+		description: "If true, will notify on start if there's a newer beta version of minikube available.",
+		set:         SetBool,
 	},
 	{
-		name: config.ReminderWaitPeriodInHours,
-		set:  SetInt,
+		name:        config.ReminderWaitPeriodInHours,
+		description: "The number of hours to wait before reminding there's a newer version of minikube again.",
+		set:         SetInt,
 	},
 	{
-		name: config.WantNoneDriverWarning,
-		set:  SetBool,
+		name:        config.WantNoneDriverWarning,
+		description: "If false, will stop recommending to use Docker driver instead of none driver.",
+		set:         SetBool,
 	},
 	{
 		name: config.WantVirtualBoxDriverWarning,
 		set:  SetBool,
 	},
 	{
-		name: config.ProfileName,
-		set:  SetString,
+		name:        config.ProfileName,
+		set:         SetString,
+		description: "The name of the minikube cluster (profile) default \"minikube\"",
```

Comment:
Would be nice to keep the consistency of having: `<description>. (default <default>)`